stardog/admin.py

```diff
@@ -956,3 +1008,107 @@ def available(self):
 
     def __repr__(self):
         return self.name
+
+
+class DataSource(object):
+    """Initializes a DataSource
+
+    See Also:
+        https://docs.stardog.com/virtual-graphs/virtual-sources
+    """
+
+    def __init__(self, name, client):
+        """Initializes a DataSource.
+
+        Use :meth:`stardog.admin.Admin.data_source`,
+        :meth:`stardog.admin.Admin.data_sources`, or
+        :meth:`stardog.admin.Admin.new_data_source` instead of
+        constructing manually.
+        """
+
+        self.data_source_name = name
+        self.path = '/admin/data_sources/{}'.format(name)
+        self.client = client
```

Comment:
The member variables that are protected should start with a _

Reply:
since all the protected member variables in the code don't start with a `_` I will create a ticket for refactoring the entire code for it, to have all classes consistent. This will be part of another PR